### PR TITLE
Reduce number of reminder_case_update_queue processes

### PR DESCRIPTION
**Pull request by rohit-dimagi**
##### SUMMARY

* Reduce number of reminder_case_update_queue processes

##### ENVIRONMENTS AFFECTED

* ICDS
##### ISSUE TYPE

- Change Pull Request

### CODE REVIEW

File: environments/icds-cas/app-processes.yml [hunk 1]
```diff
@@ -55,8 +55,8 @@ celery_processes:
       concurrency: 4
     reminder_case_update_queue:
       pooling: gevent
-      concurrency: 5
```
Comment (snopoke) [inline environments/icds-cas/app-processes.yml]: `concurrency` here means how many gevent tasks can run at once. We should leave this at 5 for all of them.

Comment (rohit-dimagi) [reply environments/icds-cas/app-processes.yml]: Ah ok.I got confused with  "it has to have the same total concurrency which  we have right now is 1330".

**Comment by emord** [comment]
I think this change may be easier to make if we started using this pattern for these celery servers: https://github.com/dimagi/commcare-cloud/pull/3179/files#r316056636

**Comment by rohit-dimagi** [comment]
Closing in favor of https://github.com/dimagi/commcare-cloud/pull/3210

**Pull request closed without merging**